charts/app/values.yaml

```diff
@@ -27,6 +27,11 @@ appealsServiceApi:
   config:
     port: 3000
     logLevel: trace
+    notify:
+      baseUrl: https://api.notifications.service.gov.uk
+      secretName: ""
+      serviceId: dummy-service-id-for-notify
```

Comment:
@frank-dodion I assume I need to ask PINS for the correct value for this?

Reply:
I think we do not need serviceId because it can be derived from a valid apiKey.